src/test/scala/cognite/spark/v1/EventsRelationTest.scala

```diff
@@ -487,6 +487,69 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
     spark.sparkContext.setLogLevel("WARN")
   }
 
+  it should "allow upsert in savemode" taggedAs WriteTest in {
+    val source = "spark-events-test"
+
+    // Cleanup events
```

Comment:
These comments look like they should be deleted or uncommented.

Reply:
these comments are copied over from the  previous test (it should support upserts), so should I remove them there as well?